Generate.py

```diff
@@ -451,7 +451,7 @@ def roll_settings(weights: dict, plando_options: PlandoOptions = PlandoOptions.b
     if "linked_options" in weights:
         weights = roll_linked_options(weights)
 
-    valid_keys = set()
+    valid_keys = {"triggers"}
```

Comment:
I'm worried about side effects from putting `"triggers"` in this `set` this early (because I was thinking of just adding it to the set right before the warning `for` loop near the end of the function).

It might be ok, but we should make sure it's tested and thought through with everything that is done with `valid_keys` before that warning loop.

Reply:
`valid_keys` gets rebuilt per player, and `roll_triggers` just adds the names that it triggers off of (fake names people use for more complex trigger setups) to the set. it should be fine, but more testing is probably a good idea